apisix/ssl/router/radixtree_sni.lua

```diff
@@ -194,6 +195,24 @@ function _M.match_and_set(api_ctx)
         end
     end
 
+    if matched_ssl.value.client then
+        local client_cert = matched_ssl.value.client.ca
```

Comment:
The name `client_cert` is confusing, actually, it's the CA cert(s) to verify the client cert. What about `ca_cert`.